### PR TITLE
Restaurant page features

### PR DESCRIPTION
**Pull request by helciodev**
The features of this project are, user can:

1. page change dynamically
2. Javascript modules.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,88 @@
...
+## Built With
+
+- [![forthebadge](https://forthebadge.com/images/badges/made-with-javascript.svg)](https://forthebadge.com)
+- [![forthebadge](https://forthebadge.com/images/badges/uses-html.svg)](https://forthebadge.com)
+- [![forthebadge](https://forthebadge.com/images/badges/uses-css.svg)](https://forthebadge.com)
+
+## Getting Started
+
+
+### Installation instructions
```
Comment (afanimartin) [inline README.md:49]: - Add instructions for installing packages, building the app locally and running

File: package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "js-restaurant-page",
+  "version": "1.0.0",
+  "description": "Restaurant page with the pages build entirely with JavaScript and Webpack for modularization",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --watch"
+  },
```
Comment (afanimartin) [inline package.json]: - Add missing commands
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

File: modules/home.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+function addClasses(elem, classes) {
```
Comment (afanimartin) [inline modules/home.js]: - Use ES6 syntax for creating functions

File: modules/home.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  classes.split(' ').forEach((clas) => elem.classList.add(clas));
+}
+
+function appendChildren(node, ...children) {
```
Comment (afanimartin) [inline modules/home.js]: - Use ES6 synatx

File: modules/contact.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { appendChildren, addClasses } from './home';
+
+const contactContainer = document.createElement('div');
+const headingOneContact = document.createElement('h1');
+
+addClasses(headingOneContact, 'text-3xl gray-500 font-bold opacity-100');
+addClasses(contactContainer, 'mx-auto w-1/2 bg-white opacity-90 px-5 py-3 text-center');
+
+headingOneContact.textContent = '222-333-444';
+appendChildren(contactContainer, headingOneContact);
```
Comment (afanimartin) [inline modules/contact.js:10]: - Implement this module using ES6 function syntax and instead export that function

File: modules/menu.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  <h3 class="text-red-400 text-2xl font-bold-font-thin">Pank magna</h3>
+  <img src="./assets/images/asian.jpg" class="h-full w-full" alt="">
+  </div>
+  <p class="p-3">Lorem ipsum dolor sit amet consectetur adipisicing elit. Earum pariatur aspernatur aut omnis nemo. Laboriosam rem beatae nulla enim saepe, vel modi amet?</p>
+  `);
+
+headingOneMenu.textContent = "Gary's menu";
+
+appendChildren(menuContainer, headingOneMenu, dish, dish2, dish3, dish4);
+export default menuContainer;
```
Comment (afanimartin) [inline modules/menu.js]: - Implement this module using ES6 syntax for functions and export that function

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work on this project. I like the design of your app.
### Well done 🌵 🌵 🌵 
- use of modules
- necessary packages installed

### Changes to make 🙅🏿 🙅🏿 🙅🏿 
- update readme with instructions for getting started locally. Check my comments.
- use Es6 syntax for creating functions

### Martin Afani
#### Slack: whiz25

File: modules/menu.js (hunk 6 above)
Comment (helciodev) [reply modules/menu.js]: thanks for the very descriptive code review!

**Review by oloomoses — APPROVED** [review]
## Congratulations, Project Aproved! :tada: :muscle: 
**Hello @helciodev ,
Well done on making the changes suggested by @afanimartin. You have passed all the requirements for this project :100:** 

Happy Codding and best wishes in your next project

_slack: Oloo Moses_

**Pull request merged**